### PR TITLE
Provision Windows machine on vsphere, default machineset values to cr…

### PR DESCRIPTION
**Pull request by miyadav**
…eate it

Locally working fine , some missed config causing test not being found , working on it ..

https://mastern-jenkins-csb-openshift-qe.cloud.paas.psi.redhat.com/job/Runner-v3/196897/console

related bug - https://bugzilla.redhat.com/show_bug.cgi?id=1875939

### CODE REVIEW

**Comment by miyadav** [comment]
thanks @liangxia , for resolving test not found ..
@jhou1  @sunzhaohua2  , PTAL 
https://mastern-jenkins-csb-openshift-qe.cloud.paas.psi.redhat.com/job/Runner-v3/197926/console

File: features/machine/machine.feature [hunk 1]
```diff
@@ -377,22 +376,39 @@ Feature: Machine features testing
...
+    And I run the :get background admin command with:
+      | resource      | machine           |
+      | w             | true              |
     Then the step should succeed
-     
+
+    And 120 seconds have passed
+    When I terminate last background process
+    Then the output should contain:
+      | Provisioned  |
```
Comment (liangxia) [inline features/machine/machine.feature:407]: Perhaps we can use below steps to replace line 399 ~ line 407, which should be more robust, and a little faster as we do not need to hard wait for 120 seconds.
```
  And I wait up to 120 seconds for the steps to pass:
  """
  When I run the :get admin command with:
    | resource | machine |
    | w        | true    |
  Then the step should succeed
  And the output should contain:
    | Provisioned  |
  """
```

Comment (miyadav) [reply features/machine/machine.feature:407]: The steps get hung if I use above , even when Provisioned is obtained , hence I used the hard coded steps .. 
          selfLink: ""
    
    [03:39:53] INFO> Exit Status: 0
    [03:39:53] INFO> Shell Commands: rm -r -f -- /root/workdir/febf767459e0-
    
    [03:39:53] INFO> Exit Status: 0
    [03:39:53] INFO> === End After Scenario: Implement defaulting machineset values for vsphere, Examples (#1) ===

1 scenario (1 passed)
18 steps (18 passed)
19m19.859s
[03:39:53] INFO> === At Exit ===


When used hard coded --
1 scenario (1 undefined)
22 steps (3 skipped, 1 undefined, 18 passed)
2m14.998s

You can implement step definitions for undefined steps with these snippets:

Then(/^the step should succeed\t    Then the step should succeed$/) do
  pending # Write code here that turns the phrase above into concrete actions
end

[03:44:07] INFO> === At Exit ===

Let me know if what we can review for this , if the way steps are used is not correct or something else ..

Comment (liangxia) [reply features/machine/machine.feature:407]: Try this,
```
  And I wait up to 120 seconds for the steps to pass:
  """
  When I run the :get admin command with:
    | resource | machine |
  Then the step should succeed
  And the output should contain:
    | Provisioned  |
  """
```

Removed the line `| w        | true    |`, as we do not need to watch in this way.

Comment (miyadav) [reply features/machine/machine.feature:405]: thanks , I works now and saves time

**Comment by miyadav** [comment]
@lxia @jhou1 @sunzhaohua2 PTAL 
https://mastern-jenkins-csb-openshift-qe.cloud.paas.psi.redhat.com/job/Runner-v3/198305/console

File: features/machine/machine.feature [hunk 2]
```diff
@@ -359,10 +359,9 @@ Feature: Machine features testing
       | failure-domain.beta.kubernetes.io/region in [<%= cb.default_region %>] |
 
   # @author miyadav@redhat.com
-  # @case_id OCP-33380
   @admin
   @destructive
-  Scenario: Implement defaulting machineset values for vsphere
+  Scenario Outline: Implement defaulting machineset values for vsphere
```
Comment (liangxia) [inline features/machine/machine.feature:364]: It's a scenario outline now, no longer specific to vsphere. Perhaps remove `for vsphere` here ?

**Pull request merged**